daml-lf/tests/scenario/stable/many-fields/Test.daml

```diff
@@ -26,7 +26,7 @@ template BigTemplate
     x090: Int, x091: Int, x092: Int, x093: Int, x094: Int, x095: Int, x096: Int, x097: Int, x098: Int, x099: Int
     x100: Int, x101: Int, x102: Int, x103: Int, x104: Int, x105: Int, x106: Int, x107: Int, x108: Int, x109: Int
     x110: Int, x111: Int, x112: Int, x113: Int, x114: Int, x115: Int, x116: Int, x117: Int, x118: Int, x119: Int
-    x120: Int, x121: Int, x122: Int, x123: Int, x124: Int--, x125: Int, x126: Int, x127: Int, x128: Int, x129: Int
```

Comment:
@remyhaemmerle-da I dropped a few more fields until we figure out stack safety for the type checker